Lombiq.Npm.Targets.targets

```diff
@@ -3,7 +3,7 @@
   <Target
     Name="NpmInstall"
     AfterTargets="AfterResolveReferences"
-    BeforeTargets="NpmDotnetPrebuild;BeforeBuild"
+    BeforeTargets="NpmDotnetPrebuild;Compile"
```

Comment:
I've documented it in the [commit message](https://github.com/Lombiq/NPM-Targets/pull/8/commits/c63710a0b1d5c5e583b153f65c73e966c07dfee4):

> Replace invalid BeforeTargets target with a valid one
> - AfterResolveReferences happens AFTER BeforeBuild, so that was never valid

Reply:
I see. Please address the second part of my comment too.

Reply:
I have tested it with Gulp Extensions in CARE, now. It works flawlessly. I honestly don't expect anything to break here.

Reply:
Please open a second PR to OSOCE just with this updated compared to dev.

Reply:
And remove it from here?

Reply:
https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/61

Reply:
No, here it's needed as well (though during merging the PRs another merge will be needed, but it's necessary for integration).